.env.example

```diff
@@ -1,16 +1,24 @@
-# Gemfile
-GEM_PG_VERSION=1.2.2
+# Database
+DB_HOST=postgis
+DB_NAME=gtt
+DB_USER=gtt
+DB_PASS=gtt
+DB_PORT=5432
 
-# docker-compose.yml - postgres
-POSTGRES_USER=gtt
-POSTGRES_PASSWORD=gtt
-POSTGRES_DB=gtt
-
-# config/configuration.yml
-SMTP_ENABLE_STARTTLS_AUTO=true # (or false)
-SMTP_ADDRESS=smtp.gmail.com
-SMTP_PORT=587
-SMTP_DOMAIN=smtp.gmail.com
-SMTP_AUTHENTICATION=plain # (or login)
+# Email
+SMTP_ADDRESS=smtp
+SMTP_DOMAIN=smtp.example.com
+SMTP_PORT=1025
 SMTP_USER_NAME=YOUR_EMAIL_ADDRESS
 SMTP_PASSWORD=YOUR_PASSWORD
+SMTP_ENABLE_STARTTLS_AUTO=true
+
+# Setup
+REDMINE_LANG=en
+REDMINE_EMAIL=gtt@example.com
+
+# System
+RAILS_ENV=production
```

Comment:
@dkastl (CC: @smellman @nobnisai)
From yesterday's internal discussion, we agreed not to support other `development` and `test` environments, so this `RAILS_ENV` line will not be necessary. (Because, current code doesn't support configuration.)

Note that plugin's test will not be available if we stick to `production` environment, so to develop plugins with tests, we need to setup another environment.